src/managers/ApplicationCommandManager.js

```diff
@@ -100,7 +100,7 @@ class ApplicationCommandManager extends CachedManager {
 
   /**
    * Creates an application command.
-   * @param {ApplicationCommandData} command The command
+   * @param {ApplicationCommandData|RESTPostAPIApplicationCommandsJSONBody} command The command
```

Comment:
You can’t just use names of interfaces from the typings here. These have to be declared as `@external` links at the end of a file and this isn’t the naming scheme we use for these external links either. I believe there is already an APIApplicationCommand though, but please double check that